spec/io/sql_data_source_spec.rb

```diff
@@ -62,5 +62,22 @@
         }.to raise_error(ArgumentError)
       end
     end
+
+    context 'with path to sqlite3 file' do
+      subject(:df) { Daru::IO::SqlDataSource.make_dataframe(db_name, query) }
+      it { is_expected.to be_a(Daru::DataFrame) }
+      it { expect(df.nrows).to eq 2 }
+      it { expect(df.row[0][:id]).to eq 1 }
+      it { expect(df.row[0][:name]).to eq 'Homer' }
```

Comment:
Well, I'd say better way to write this would be:
```ruby
its(:nrows) { is_expected.to eq 2 }
its(:'rows.first') { is_expected.to eq {id: 1, name: 'Homer'} }
```

But it is a neatpeaking. I thinkg "enchance specs quality would be separated task, so we are good here.